Source/LoadByOS/WindowsConfigApp/Program.cs

```diff
@@ -1,18 +1,7 @@
-using AppScaffolding;
-
 namespace WindowsConfigApp
 {
-    class Program : OSConfigBase
+    class Program
     {
-        public override Type InteropFunctionsType => typeof(WinInterop);
-        public override Type[] ReferencedTypes => new Type[]
-        {
-            typeof(Bitmap),
-            typeof(Dinah.Core.WindowsDesktop.GitClient),
-            typeof(Accessibility.IAccIdentity),
-            typeof(Microsoft.Win32.SystemEvents)
-        };
```

Comment:
This was necessary when we were executing the config app to get a list of its process modules, but we're not doing that so we don't need the runtime to load the types. In fact, we don't need the launcher exes at all which is why I modified the workflows to delete them.

Reply:
As a reminder, the only reason we do not have to execute the config app to load its dependencies is because we are publishing it as self-contained, so all dependencies are in the same folder and we can just load them by file name. If we ever published not self-contained (framework dependent), we would need the run time to locate the assemblies for us because they'll be somewhere in program files or system.

This is as much a reminder to me as it is to you.

Reply:
> If we ever published not self-contained

Seems like this could be left in-code. Move to an un-called method and put these comments in it. Otherwise if we do ever need it, I'll be going: oh crap, I remember we removed in one of those PRs -- now how did we do this again...?

Reply:
I didn't remove it from the _Demo projects. Is that in-code enough? I'll add my comments in this thread to the code too.

Reply:
Oh yeah -- great point